src/sortStudents.ts

```diff
@@ -1,16 +1,75 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+const findAverageGrade = (arr: number[]): number => {
+  return arr.reduce((acc, num) => acc + num, 0) / arr.length;
+};
```

Comment:
Pay attention to the variables naming
```suggestion
const calculateAverageGrades = (grades: number[]): number => {
  return arr.reduce((acc, num) => acc + num, 0) / arr.length;
};
```